content/guidelines/testing-guidelines.md

```diff
@@ -68,7 +68,58 @@ to do anything to get new functionality in the DSC modules.
 
 ### Opt-Out from Tests
 
-Not yet written.
+Default is that all the tests are opted-in. If a test cannot be resolved
+to pass then there are different scenarios supported to opt-out from tests.
+Preferably you should opt-out from tests.
+
+- Using the opt-out JSON file `.MetaTestOptOut.json`.
+    ```yaml
+    [
+        "Common Tests - New Error-Level Script Analyzer Rules"
+    ]
+    ```
+- Configuring the `build.yaml` with the tag `ExcludeTag` under either the
+  key `Pester` or `DscTest` (JSON files override excludes for the key
+  `DscTest`).
+  ```yaml
+  Pester:
+    ExcludeTag:
+      - 'TagOnUnitTest'
+
+  DscTest:
+    ExcludeTag:
+      - Common Tests - New Error-Level Script Analyzer Rules
+      - Common Tests - Validate Example Files
+      - Common Tests - Relative Path Length
+  ```
+- Using the opt-in JSON file `.MetaTestOptIn.json`. *This is only available*
+  *for backwards compatibility, we do not recommend using this file but*
+  *instead move to the other two options.*
+  ```yaml
+  [
+    "Common Tests - Validate Example Files",
+    "Common Tests - Relative Path Length",
+  ]
+  ```
+
+#### Default opt-outs
+
+##### Test "New Error-Level Script Analyzer Rules"
+
+This HQRM tests is opt-out by default since it tests the ScriptAnalyzer
```

Comment:
I should fix that in the config file of DscResource.Test.
Will try to do tomorrow.

Reply:
Leaving it in for now and we remove it when it changes.